sketcherMinimizer.cpp

```diff
@@ -8,19 +8,25 @@
  */
 
 #include "sketcherMinimizer.h"
-#include "sketcherMinimizerMaths.h"
 
+#include <algorithm>
+#include <cstdio>
+#include <queue>
+#include <stack>
+
+#ifdef BUNDLE_TEMPLATES
```

Comment:
I think we should consider having two different options:
`BUNDLE_TEMPLATES` will bundle the default templates into the code so that you don't *need* to look for the template file in order to use coordgen.
`INCLUDE_MAEPARSER` controls whether or not the MAE parser is available. This could be useful in a future version where client-code can provide templates.

Reply:
Ricardo suggested _always_ bundling.

I think that dovetails with your suggestion, so that we'd do:

* always bundle the default templates
* if compiled with maeparser support, also allow override of template file.

So then the only define would be INCLUDE_MAEPARSER (on by default). What do you think of that?

Reply:
Makes sense